arangod/Aql/RestAqlHandler.h

```diff
@@ -109,7 +110,8 @@ class RestAqlHandler : public RestVocbaseBaseHandler {
                         std::shared_ptr<arangodb::velocypack::Builder> const& options,
                         std::shared_ptr<transaction::Context> const& ctx,
                         double const ttl, aql::SerializationFormat format,
-                        bool& needToLock, arangodb::velocypack::Builder& answer);
+                        bool& needToLock, arangodb::velocypack::Builder& answer,
+                        RebootId rebootId, std::string coordinatorId);
```

Comment:
Can be passed by const reference, but this is an unimportant optimization here.
```suggestion
                        RebootId rebootId, std::string const& coordinatorId);
```

Reply:
done